cortex/utils.py

```diff
@@ -152,11 +165,18 @@ def generate_token(config, validity=2):
             "aud": config.get("audience"),
             "sub": config.get("username"),
         }
+
+        server_ts_dt = datetime.fromtimestamp(
```

Comment:
Not a big deal but 
datetime.fromtimestamp(timestamp, tz=timezone.utc).

^^ Can use this over adding a new dependency ..

Reply:
done